cpp/ql/lib/semmle/code/cpp/security/InvalidPointerDereference/RangeAnalysisUtil.qll

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Since we have similar "stuff" in two other MCTV queries, I wonder if this should maybe be part of the range analysis library?

Reply:
I think that would be a really good idea, yes. I'll create an issue for this for when we start working on the `cpp/constant-array-overflow` query again.